app/Jobs/ApplicationDeploymentJob.php

```
@@ -2463,35 +2741,189 @@ private function add_build_env_variables_to_dockerfile()
+        $fromIndex = $dockerfile->search(function ($line) {
+            return str_starts_with(trim(strtoupper($line)), 'FROM');
+        });
+
+        if ($fromIndex !== false) {
+            // Insert comments after FROM statement
+            foreach (array_reverse($secretsComment) as $comment) {
+                $dockerfile->splice($fromIndex + 1, 0, [$comment]);
             }
         }
+
         $dockerfile_base64 = base64_encode($dockerfile->implode("\n"));
```

PHPMD (2.15.0)

The variable $dockerfile_base64 is not named in camelCase. (undefined)

(CamelCaseVariableName)